lib/data/changelog-urls.json

```diff
@@ -24,6 +24,7 @@
     "uwsgi": "https://uwsgi-docs.readthedocs.io/en/latest/#release-notes"
   },
   "docker": {
+    "adminer": "https://github.com/vrana/adminer/blob/master/CHANGELOG.md",
```

Comment:
did you tried to upstream first?

Reply:
Sorry, I don't understand what you mean with this.

Looking at the [documentation on changelogs](https://docs.renovatebot.com/key-concepts/changelogs/) would it be better to add it to source-urls instead? The repo does maintain releases and includes the changes for each release there as well: https://github.com/vrana/adminer/releases

Reply:
Ah, I think you mean adding the label to the image: https://docs.renovatebot.com/modules/datasource/docker/#description

Reply:
yes, adding the source label to the image would be preferred